<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>31</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>100</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>100</y>
     <width>31</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_3">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>100</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_6">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>250</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_5">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>250</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_9">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>400</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_8">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>400</y>
     <width>185</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>65</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>89</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>554</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>554</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>554</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>8</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>26</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="x_score">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>8</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>8</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QLabel" name="o_score">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>8</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>26</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
